usecases/schema/class_test.go

```diff
@@ -73,28 +73,6 @@ func Test_AddClass(t *testing.T) {
 		assert.EqualError(t, err, "'' is not a valid class name")
 	})
 
-	t.Run("with permuted-casing class names", func(t *testing.T) {
```

Comment:
is it intended to remove it ?

Reply:
Yes it is, since the validation is now done in cluster/store